setup.py

```diff
@@ -10,7 +10,7 @@
     subprocess.check_output("git describe --always".split()).strip().decode("ascii")
 )
 TESTS_REQUIRE = ["pytest-cov", "pytest-socket", "tox"]
-DEV_REQUIRE = ["black", "mypy", "pylint", "safety"]
+DEV_REQUIRE = ["black", "mypy", "pylint", "safety", "pandas"]
```

Comment:
Why do we need a monster of pandas as a dependency? Only for csv, or do I miss something? https://docs.python.org/3.6/library/csv.html